Segfault when using -G switch on yay-git

#### Affected Version
yay v9.1.0.r10.g8ebc41d - libalpm v11.0.2

#### Issue
`-G` results in segfault

<!-- The following sections may be left out if not relevant -->
#### Steps to reproduce
<!-- Use exact commands where applicable -->
1. yay -G libreoffice-fresh

#### Output
````
[parker@stealth build]$ yay -G libreoffice-fresh
:: Querying AUR...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x560fd91c8ba4]

goroutine 1 [running]:
github.com/jguer/go-alpm.(*DB).Name(0x0, 0x0, 0x0)
        /home/parker/.cache/yay/yay-git/src/yay/vendor/github.com/jguer/go-alpm/db.go:101 +0x24
main.getPkgbuildsfromABS(0xc000081390, 0x1, 0x1, 0xc000018064, 0x12, 0x0, 0x0, 0x0)
        /home/parker/.cache/yay/yay-git/src/yay/download.go:240 +0x951
main.getPkgbuilds(0xc000080c60, 0x1, 0x1, 0x560fd93a3aac, 0xc0000d9f18)
        /home/parker/.cache/yay/yay-git/src/yay/download.go:154 +0xdab
main.handleGetpkgbuild(0x560fd96e2a07, 0x1)
        /home/parker/.cache/yay/yay-git/src/yay/cmd.go:237 +0x45
main.handleCmd(0x0, 0x0)
        /home/parker/.cache/yay/yay-git/src/yay/cmd.go:160 +0x17d
main.main()
        /home/parker/.cache/yay/yay-git/src/yay/main.go:222 +0xd5
````
````
{
        "aururl": "https://aur.archlinux.org",
        "buildDir": "$HOME/.cache/yay",
        "editor": "",
        "editorflags": "",
        "makepkgbin": "makepkg",
        "makepkgconf": "",
        "pacmanbin": "pacman",
        "pacmanconf": "/etc/pacman.conf",
        "tarbin": "bsdtar",
        "redownload": "no",
        "rebuild": "no",
        "answerclean": "",
        "answerdiff": "",
        "answeredit": "",
        "answerupgrade": "",
        "gitbin": "git",
        "gpgbin": "gpg",
        "gpgflags": "",
        "mflags": "",
        "sortby": "votes",
        "gitflags": "",
        "removemake": "ask",
        "requestsplitn": 150,
        "sortmode": 0,
        "completionrefreshtime": 7,
        "sudoloop": false,
        "timeupdate": false,
        "devel": true,
        "cleanAfter": false,
        "gitclone": true,
        "provides": true,
        "pgpfetch": true,
        "upgrademenu": true,
        "cleanmenu": true,
        "diffmenu": true,
        "editmenu": false,
        "combinedupgrade": false,
        "useask": false
}
````

I have encountered the same problem, this may be related to https://github.com/Jguer/yay/commit/895788e01b62cc69007680c0cf686568546e8d60.
```go
GOTRACEBACK=crash yay -G rtl8812au-dkms-git
:: Querying AUR...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x4cfa02]

goroutine 1 [running]:
panic(0x6f1120, 0x973140)
        /usr/lib/go/src/runtime/panic.go:556 +0x2cb fp=0xc0000e9850 sp=0xc0000e97c0 pc=0x430c4b
runtime.panicmem()
        /usr/lib/go/src/runtime/panic.go:82 +0x5e fp=0xc0000e9870 sp=0xc0000e9850 pc=0x42fc8e
runtime.sigpanic()
        /usr/lib/go/src/runtime/signal_unix.go:390 +0x182 fp=0xc0000e98c0 sp=0xc0000e9870 pc=0x445c42
github.com/jguer/go-alpm.(*DB).Name(0x0, 0x0, 0x0)
        /build/yay-git/src/yay/vendor/github.com/jguer/go-alpm/db.go:101 +0x22 fp=0xc0000e98e8 sp=0xc0000e98c0 pc=0x4cfa02
main.getPkgbuildsfromABS(0xc000090eb0, 0x1, 0x1, 0xc000018084, 0x38, 0x0, 0x0, 0x0)
        /build/yay-git/src/yay/download.go:240 +0x94f fp=0xc0000e9c10 sp=0xc0000e98e8 pc=0x66cdff
main.getPkgbuilds(0xc000090c30, 0x1, 0x1, 0x74690c, 0xc0000e9f18)
        /build/yay-git/src/yay/download.go:154 +0xda9 fp=0xc0000e9e88 sp=0xc0000e9c10 pc=0x66c409
main.handleGetpkgbuild(0x981a07, 0x1)
        /build/yay-git/src/yay/cmd.go:237 +0x43 fp=0xc0000e9ec0 sp=0xc0000e9e88 pc=0x65a993
main.handleCmd(0x0, 0x0)
        /build/yay-git/src/yay/cmd.go:160 +0x17b fp=0xc0000e9f38 sp=0xc0000e9ec0 pc=0x6599cb
main.main()
        /build/yay-git/src/yay/main.go:222 +0xd3 fp=0xc0000e9f98 sp=0xc0000e9f38 pc=0x680183
runtime.main()
        /usr/lib/go/src/runtime/proc.go:201 +0x207 fp=0xc0000e9fe0 sp=0xc0000e9f98 pc=0x432977
runtime.goexit()
        /usr/lib/go/src/runtime/asm_amd64.s:1333 +0x1 fp=0xc0000e9fe8 sp=0xc0000e9fe0 pc=0x45d4d1

goroutine 2 [force gc (idle)]:
runtime.gopark(0x762068, 0x9a8af0, 0x1410, 0x1)
        /usr/lib/go/src/runtime/proc.go:302 +0xeb fp=0xc000044f80 sp=0xc000044f60 pc=0x432d6b
runtime.goparkunlock(0x9a8af0, 0x1410, 0x1)
        /usr/lib/go/src/runtime/proc.go:308 +0x53 fp=0xc000044fb0 sp=0xc000044f80 pc=0x432e13
runtime.forcegchelper()
        /usr/lib/go/src/runtime/proc.go:251 +0xb3 fp=0xc000044fe0 sp=0xc000044fb0 pc=0x432be3
runtime.goexit()
        /usr/lib/go/src/runtime/asm_amd64.s:1333 +0x1 fp=0xc000044fe8 sp=0xc000044fe0 pc=0x45d4d1
created by runtime.init.4
        /usr/lib/go/src/runtime/proc.go:240 +0x35

goroutine 3 [GC sweep wait]:
runtime.gopark(0x762068, 0x9a8c20, 0x42140c, 0x1)
        /usr/lib/go/src/runtime/proc.go:302 +0xeb fp=0xc000045780 sp=0xc000045760 pc=0x432d6b
runtime.goparkunlock(0x9a8c20, 0x7a140c, 0x1)
        /usr/lib/go/src/runtime/proc.go:308 +0x53 fp=0xc0000457b0 sp=0xc000045780 pc=0x432e13
runtime.bgsweep(0xc00006c000)
        /usr/lib/go/src/runtime/mgcsweep.go:52 +0x8f fp=0xc0000457d8 sp=0xc0000457b0 pc=0x425fef
runtime.goexit()
        /usr/lib/go/src/runtime/asm_amd64.s:1333 +0x1 fp=0xc0000457e0 sp=0xc0000457d8 pc=0x45d4d1
created by runtime.gcenable
        /usr/lib/go/src/runtime/mgc.go:216 +0x58

goroutine 18 [finalizer wait]:
runtime.gopark(0x762068, 0x9c6a00, 0x140f, 0x1)
        /usr/lib/go/src/runtime/proc.go:302 +0xeb fp=0xc000040728 sp=0xc000040708 pc=0x432d6b
runtime.goparkunlock(0x9c6a00, 0x140f, 0x1)
        /usr/lib/go/src/runtime/proc.go:308 +0x53 fp=0xc000040758 sp=0xc000040728 pc=0x432e13
runtime.runfinq()
        /usr/lib/go/src/runtime/mfinal.go:175 +0x99 fp=0xc0000407e0 sp=0xc000040758 pc=0x41daf9
runtime.goexit()
        /usr/lib/go/src/runtime/asm_amd64.s:1333 +0x1 fp=0xc0000407e8 sp=0xc0000407e0 pc=0x45d4d1
created by runtime.createfing
        /usr/lib/go/src/runtime/mfinal.go:156 +0x61
[1]    12063 abort (core dumped)  GOTRACEBACK=crash yay -G rtl8812au-dkms-git
```

--- a/go.mod
+++ b/go.mod
@@ -3,6 +3,6 @@ module github.com/Jguer/yay/v9
 require (
 	github.com/Morganamilo/go-pacmanconf v0.0.0-20180910220353-9c5265e1b14f
 	github.com/Morganamilo/go-srcinfo v0.0.0-20180702014333-368edc79b2c5
-	github.com/jguer/go-alpm v0.0.0-20190214204021-327c7630865b
+	github.com/jguer/go-alpm v0.0.0-20190302211415-9c82d5170ce0
 	github.com/mikkeloscar/aur v0.0.0-20181111113612-b71516da3ae9
 )

